### PR TITLE
Restructure docs v1

### PR DESCRIPTION
**Pull request by schimih**
#### Did you test the changes locally ?
- [x] yes
- [ ] no

#### Which category (categories) does this pull request belong to?
- [ ] document new feature
- [ ] update documentation that is not relevant anymore
- [ ] add examples or more information about a component
- [ ] fix grammar issues
- [x] other

### CODE REVIEW

File: docs/learn/EGLD.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+id: EGLD
+title: What is EGLD?
```
Comment (andreibancioiu) [inline docs/learn/EGLD.md:3]: _What is MultiversX?_

**Review by andreibancioiu — DISMISSED** [review]

File: docs/learn/EGLD.md (hunk 1 above)
Comment (bogdan-rosianu) [reply docs/learn/EGLD.md:3]: 👍

File: docs/learn/EGLD.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+id: EGLD
```
Comment (bogdan-rosianu) [inline docs/learn/EGLD.md:2]: id: multiversx

**Review by bogdan-rosianu — COMMENTED** [review]
also add redirects from the old pages to the new pages

**Review by andreibancioiu — APPROVED** [review]

**Review by sstanculeanu — APPROVED** [review]

**Pull request merged**